### ISSUE TITLE
small warning testing latest Pillow 3.1.0 on windows

### ISSUE THREAD

**Issue by stonebig** [python-pillow/Pillow#1642]
see https://github.com/scipy/scipy/issues/5666

```
Warning (from warnings module):
  File "D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\PIL\Image.py", line 842
    self.load()
ResourceWarning: unclosed file <_io.BufferedReader name='D:\\WinPython\\basedir34\\buildQt5\\winpython-3.4.4.amd64\\python-3.4.4.amd64\\lib\\site-packages\\scipy\\misc\\tests\\data\\foo3x5x4indexed.png'>
```

**Comment by wiredfool** [comment]
This appears to be related to #835

**Comment by aclark4life** [comment]
@wiredfool But not a duplicate of #835?

**Comment by hugovk** [comment]
Can no longer reproduce with SciPy 0.19.1, NumPy 1.13.1 and Pillow 4.2.1 on macOS:
```
[hugo:/tmp] 5s % python3
Python 3.6.0 (default, Mar  4 2017, 12:32:34)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import scipy;scipy.test()
Running unit tests for scipy
NumPy version 1.13.1
NumPy relaxed strides checking option: True
NumPy is installed in /usr/local/lib/python3.6/site-packages/numpy
SciPy version 0.19.1
SciPy is installed in /usr/local/lib/python3.6/site-packages/scipy
Python version 3.6.0 (default, Mar  4 2017, 12:32:34) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
nose version 1.3.7
/usr/local/Cellar/python3/3.6.0_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/importlib/_bootstrap.py:205: ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__
  return f(*args, **kwds)
..........................................................................................................................................................................................................................................................................................K.....................................................................................................................................................................................................................................................................................................................................K.............................................................................too many axes: 2 (effrank=2), expected rank=1
too many axes: 2 (effrank=2), expected rank=1
.....K........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................./usr/local/lib/python3.6/site-packages/scipy/linalg/basic.py:1018: RuntimeWarning: internal gelsd driver lwork query error, required iwork dimension not returned. This is likely the result of LAPACK bug 0038, fixed in LAPACK 3.2.2 (released July 21, 2010). Falling back to 'gelss' driver.
  warnings.warn(mesg, RuntimeWarning)
....................................................................................................................0-th dimension must be fixed to 3 but got 15
..0-th dimension must be fixed to 3 but got 5
..............0-th dimension must be fixed to 4 but got 2
0-th dimension must be fixed to 4 but got 2
.0-th dimension must be fixed to 4 but got 2
0-th dimension must be fixed to 4 but got 2
.0-th dimension must be fixed to 4 but got 2
0-th dimension must be fixed to 4 but got 2
0-th dimension must be fixed to 4 but got 2
0-th dimension must be fixed to 4 but got 2
0-th dimension must be fixed to 4 but got 2
0-th dimension must be fixed to 4 but got 2
.............S....................K.........................................................................................................................................................................................................................................................................................................................K................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................S.....................K............................................K.........K....K....K.KK...........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................K...................................................Optimization terminated successfully.    (Exit mode 0)
            Current function value: 0.0
            Iterations: 1
            Function evaluations: 5
            Gradient evaluations: 1
.K............................................................................................................................................................................................S.K....S............................................................................................................................................................................................................................................................................................................................./usr/local/lib/python3.6/site-packages/scipy/signal/filter_design.py:1452: BadCoefficients: Badly conditioned filter coefficients (numerator): the results may be meaningless
  "results may be meaningless", BadCoefficients)
.........................................................................................................................................................................................................................................SS.............................................................................................K.......................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................SS..............S.....S...SSSSS.SSSSSSSSSS.....SSS..S.....S....K................S...S............SSSSSKK...SSSS..S.S...........S.........SS............S.....S...SSSSS.SSSSSSSSSS.....SSS........S....K................S...S............SSSS....SSSS..S.S..............S...................................S......................................................S.................S.............S..............S.....................................S........................................................S.................S.............S..............S........S..SS.........S....S...SSSSS.SSSSSSSSSS.....SSS..S.....S....K...........SSS..S...S............SSSS.....SSSS..S.S...........S.......S........................S.........................S......................SSS...S...............................S..............S......S......................S.....................................................SSS..S.................................S.............S.........SS..............S.....S...SSSSS.SSSSSSSSSS.....SSS..S.....S....K................S...S............SSSSSKK...SSSS..S.S...........S.........SS............S.....S...SSSSS.SSSSSSSSSS.....SSS........S....K................S...S............SSSS....SSSS..S.S..............S...................................S......................................................S.................S.............S..............S.....................................S........................................................S.................S.............S..............S........S..SS.........S....S...SSSSS.SSSSSSSSSS.....SSS..S.....S....K...........SSS..S...S............SSSS.....SSSS..S.S...........S.......S........................S.........................S......................SSS...S...............................S..............S......S......................S.....................................................SSS..S.................................S.............S.........SS..............S.....S...SSSSS.SSSSSSSSSS.....SSS..S.....S....K................S...S............SSSSSKK...SSSS..S.S...........S.........SS............S.....S...SSSSS.SSSSSSSSSS.....SSS........S....K................S...S............SSSS....SSSS..S.S..............S...................................S......................................................S.................S.............S..............S.....................................S........................................................S.................S.............S..............S........S..SS.........S....S...SSSSS.SSSSSSSSSS.....SSS..S.....S....K...........SSS..S...S............SSSS.....SSSS..S.S...........S.......S........................S.........................S......................SSS...S...............................S..............S......S......................S.....................................................SSS..S.................................S.............S.........SS..............S.....S...SSSSS.SSSSSSSSSS.....SSS..S.....S....K................S...S............SSSSSKK...SSSS..S.S...........S.........SS............S.....S...SSSSS.SSSSSSSSSS.....SSS........S....K................S...S............SSSS....SSSS..S.S..............S...................................S......................................................S.................S.............S..............S.....................................S........................................................S.................S.............S..............S........S..SS.........S....S...SSSSS.SSSSSSSSSS.....SSS..S.....S....K...........SSS..S...S............SSSS.....SSSS..S.S...........S.......S........................S.........................S......................SSS...S...............................S..............S......S......................S.....................................................SSS..S.................................S.............S.........SS..............S....S...SSSSS.SSSSSSSSSS.....SSS..S.....S....K................S...S............SSSSSKK...SSSS..S.S...........S.........SS............S.....S...SSSSS.SSSSSSSSSS.....SSS........S....K................S...S............SSSS....SSSS..S.S..............S...................................S......................................................S.................S.............S..............S.....................................S........................................................S.................S.............S..............S........S..SS.........S....S...SSSSS.SSSSSSSSSS.....SSS..S.....S....K...........SSS..S...S............SSSS.....SSSS..S.S...........S.......S........................S.........................S......................SSS...S...............................S..............S......S......................S.....................................................SSS..S.................................S.............S...............SSSSSSSSSSSSSSSSSSSSSSSSSSSSS............................S.....S...................SSSS.SSSSSSSSSS....................SSS..S.......................S....K.................................................................................................................S..................S..................S.SSSKK...SSSS....S............................................................................................................................................S.........S.......................SSSSSSSSSSSSSSSSSSSSSSSSSSSSS..S.........KS.....SS.................KSSSS.SSSSSSSSSS....................SSS..S.......................S....K.................................................................................................................S..................S..................S.SSSKK...SSSS....S............................................................................................................................................SK..............SSSSSSSSSSSSSSSSSSSSSSSSSSSSS..........................S.....S..SSSSSSSSSSSSSSSS.SSSS.SSSSSSSSSS....SSSSSSSSSSSSSSSSSSS....SSSSSSSSSSSSSSSS......S....KSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS.................................................................................SSSSSSSSSSSSSSSSS..S..................SSSS....SSSS....S...............................................................................................................................................S.........S.......................SSSSSSSSSSSSSSSSSSSSSSSSSSSSS..S........S.....SS.SSSSSSSSSSSSSSSS.SSSS.SSSSSSSSSS....SSSSSSSSSSSSSSSSSSS..S.SSSSSSSSSSSSSSSS......S....KSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS.................................................................................SSSSSSSSSSSSSSSSS..S..................SSSS.....SSSS....S...............................................................................................................................................SK...................SSSSSSSSSSSSSSSSSSSSSSSSSSS........................................................................................................................................................................................................................................................................................S.......................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................S..........S.............................SSSSSSSSSSSSSSSSSSSSSSSSSSS..S.................S.......................................................................................S...........................K................................................................................................................................S......................................................................................................................................................................................................................................................................................................................................................K................................................................................................................................................S.K....................SSSSSSSSSSSSSSSSSSSSSSSSSSS..........................................................................................................................................................................................................................................................................................S.......................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................S..........S.............................SSSSSSSSSSSSSSSSSSSSSSSSSSS..S.................S.......................................................................................S.............................K................................................................................................................................S......................................................................................................................................................................................................................................................................................................................................................K................................................................................................................................................S.K............S..SSSSSSSSSSSSSSSSSSSSSSSSSSSSS.......................S....S..SSSSSSSSSSSSSSSS.SSSS.SSSSSSSSSS....SSSSSSSSSSSSSSSSSSS..S.SSSSSSSSSSSSSSSS......S....KSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS..........................................................................SSS....SSSSSSSSSSSSSSSSS..S..................SSSS.....SSSS....S............................................................................................................................................S.........S.....................S....SSSSSSSSSSSSSSSSSSSSSSSSSSS..SSSSSSSSSSSSSSSS...............SSSSSSSSSSSSSSSS..................................SSSSSSSSSSSSSSSS...................S.SSSSSSSSSSSSSSSS............SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS..........................................................................SSS.....SSSSSSSSSSSSSSSSS...................................................................................................................................................................................................................................................................................................................................................................................................................................................................................SSSSSSSSSSSSSSSSSSSS...........S....SSSSSSSSSSSSSSSSSSSSSSSSSSS...............................SSSSSSSSSSSSSSSS...................................SSSSSSSSSSSSSSSS.....................SSSSSSSSSSSSSSSS.............SSSSSSSSSSSSSSSS...SSSSSSSSSSSSSSSS..........................................................................SSS....SSSSSSSSSSSSSSSSS....................................................................................................................................................................................................................................................................................................................................................................................................................................................................................SSSSSSSSSSSSSSSSSSSS............................................................................S...S............../usr/local/lib/python3.6/site-packages/scipy/spatial/_plotutils.py:20: MatplotlibDeprecationWarning: The ishold function was deprecated in version 2.0.
  was_held = ax.ishold()
................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................K......................................................................................................................................................................................................................................................................................................................................................................................KKKKK..KK..KK.............................................................................................................KKKK..K........KKKK........................................................................................................................................................K.K.............................K.....K..K...K....................K..............................SSSSSSSSSSSSSSSSSS/usr/local/lib/python3.6/site-packages/scipy/special/tests/test_orthogonal.py:273: FutureWarning: In the future extra properties will not be copied across when constructing one poly1d from another
  assert_almost_equal(p(0.315), np.poly1d(p)(0.315), err_msg=pstr)
........................./usr/local/lib/python3.6/site-packages/scipy/special/tests/test_orthogonal_eval.py:60: FutureWarning: In the future extra properties will not be copied across when constructing one poly1d from another
  poly = np.poly1d(cls(*p))
................................SSSSS...................................................................................................................................................................................................S................S.......................................................................................S................S................S................S.................S......................................................................................S..........................................................................................................................S...................................S.................S....................................................S.................S.................S........................................................................................................................................................................S.............................................................................................................................................................................S................S..................S........................................................................S................S................S................S................S.....................................................S........................................................................................S................S................S......................................................................K................K................S.............................................................................................................S..............................SS.....................................................................................................................................................................................................................................................................................................................................................................K..........................................................................K.......................................................................................................................................S.................................................................................................................................................................................................................................................................................................................................................................................................................S..S................................................./usr/local/lib/python3.6/site-packages/scipy/stats/mstats_basic.py:2486: MaskedArrayFutureWarning: In the future the default for argsort will be axis=-1, not the current None, to match its documentation and np.argsort. Explicitly pass -1 or None to silence this warning.
  (n + 1.0 - alpha - beta))
......................................................................................................................................................................................................................................................................................../usr/local/lib/python3.6/site-packages/scipy/stats/stats.py:4853: DeprecationWarning: Calling `mannwhitneyu` without specifying `alternative` is deprecated.
  "`alternative` is deprecated.", DeprecationWarning)
.....K........................................................................................................................................................................................................................................S..S.SSSSSS.S..S..S..S..S..S..S.SSSSSS.S..S..S..S..S..S..S.SSSSSS.S..S..S..S..S..S..S.SSSSSS.S..S..S..S..S..S.SSS.S..S..S.SSS.S..S..S.SSS.S..S..S.SSS.S..S.........
----------------------------------------------------------------------
Ran 24207 tests in 518.374s

OK (KNOWNFAIL=91, SKIP=1896)
<nose.result.TextTestResult run=24207 errors=0 failures=0>
>>>
```
See also https://github.com/python-pillow/Pillow/issues/835#issuecomment-331670254.
